spec/rubocop/cop/style/method_call_with_args_parentheses_spec.rb

```diff
@@ -631,5 +648,26 @@ def seatle_style arg = default(42)
         expect(autocorrect_source(original)).to eq(original)
       end
     end
+
+    context 'allowing parens in camel-case methods' do
+      let(:cop_config) do
+        {
+          'EnforcedStyle' => 'omit_parentheses',
+          'AllowParenthesesInCamelCaseMethod' => true
+        }
+      end
+
+      it 'accepts parens for camel-case method names' do
+        expect_no_offenses('Array(nil)')
+      end
+
+      it 'does not auto-correct' do
```

Comment:
Testing auto-correct here is redundant since `autocorrect` is only invoked on offenses.

Reply:
This was put here to match the similar example above for the `AllowParenthesesInMultilineCall` cases (lines 624ff). But I can remove it if you prefer.

Reply:
Removed.